docs/pages/deploy-a-cluster/deployments/gcp.mdx

```diff
@@ -80,34 +68,36 @@ see  [Admin Guide: Troubleshooting](../../management/admin/troubleshooting.mdx)
 
 ### Storage: Cloud Firestore
 
-[Cloud Firestore](https://cloud.google.com/firestore/) This storage backend uses real-time
-updates to keep individual auth instances in sync and requires Firestore configured
+The [Firestore](https://cloud.google.com/firestore/) backend uses real-time
+updates to keep individual Auth Servers in sync, and requires Firestore configured
 in native mode.
 
-Add this storage configuration in teleport section of the config file (by default it's `/etc/teleport.yaml`):
+To configure Teleport to store audit events in Firestore, add the following to
+the teleport section of your Auth Server's config file (by default it's `/etc/teleport.yaml`):
 
 ```yaml
 teleport:
   storage:
     type: firestore
     collection_name: cluster-data
-    project_id: EXAMPLE_gcp-proj-with-firestore-enabled
-    credentials_path: /var/lib/teleport/firestore_creds.json
-    audit_events_uri: ['firestore://events_table_name?projectID=example_Teleport-Project-Name&credentialsPath=/var/lib/teleport/gcs_creds.json']
+    project_id: <Var name="GCP-Project-ID" description="Your GCP project ID"/>
```

Comment:
Note: waiting on docs team to add a `default` attribute here.

cc @ptgott

Reply:
Ok, I'll remove the use of Var throughout this page and just use generic placeholders like we had before.

Reply:
All set. Thanks Paul!